index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <link rel="stylesheet" href="index.css">
+    </head>
+    <body>
+        <header class="wrapper">
+            <div class="box a">LOGO</div>
+            <div class="box b">
+                <nav>
+                    <ul class="header">
+                        <li>HOME</li>
+                        <li>ABOUT</li>
+                        <li class="gallery">GALLERY</li>
+                        <li>LOG IN</li>
+                    </ul>
+                </nav>
+            </div>
+            <aside class="box c">
+                    <div>MENU</div>
+                    <div class="aside-bottom">
```

Comment:
you have a list of items here. This it is a `ul`. Also it looks like it is navigation. So the `nav` tag would fit here perfectly